chore: gds flow doc (#589)
FPGA-Research-Manchester/FABulous: docs/source/user_guide/cli_doc/cli_feature.md

# FABulous CLI — Features and Usage

In FABulous, we provides two primary method to drive the tool. First is the interactive REPL mode, which is good for performing design space exploration. Second is the batch mode, which is good for CI and automation. This document summarizes the key features of both modes and provides guidance on when to use each.

## Interactive mode

To start the REPL, simply run `FABulous start` and this will lunch the interactive shell in the current working directory.

```bash
FABulous start

# you will see a prompt like this:
FABulous>
```

> But some times you might want to start the REPL in for a sepecific project directory, and you can do this by providing the `--project-dir/-p` arguments to chooses which project directory to load.

Reviewer: ```suggestion
If you want to start the REPL in for a sepecific project directory, and you can do this by providing the `--project-dir/-p` arguments to chooses which project directory to load.
```
